pkg/oc/cli/admin/release/new.go

```diff
@@ -47,12 +47,12 @@ func NewNewOptions(streams genericclioptions.IOStreams) *NewOptions {
 func NewRelease(f kcmdutil.Factory, parentName string, streams genericclioptions.IOStreams) *cobra.Command {
 	o := NewNewOptions(streams)
 	cmd := &cobra.Command{
-		Use:   "new",
+		Use:   "new [SRC=DST ...]",
 		Short: "Create a new OpenShift release",
 		Long: templates.LongDesc(`
 			OpenShift uses long-running active management processes called "operators" to
-			keep the cluster running and manage component lifecycle. This command assists
-			composing a set of images and operator definitions into a single update payload
+			keep the cluster running and manage component lifecycle. This command
+			composes a set of images and operator definitions into a single update payload
```

Comment:
I can pull this orthogonal change out into its own PR if that helps with review.

Reply:
It’s fine